src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -141,6 +142,17 @@ function WorkspaceMoreFeaturesPage({policy, route}: WorkspaceMoreFeaturesPagePro
         [isSmallScreenWidth, styles, renderItem, translate],
     );
 
+    function fetchFeatures() {
```

Comment:
do we prefer this or `const fetchFeatures = () => {}` appraoch?

Reply:
We don't have a define guideline for that, do you want me to change to `const...`?